cmd/boulder-va/main.go

```diff
@@ -85,10 +85,17 @@ func main() {
 	}
 
 	var servers bdns.ServerProvider
-	servers, err = bdns.StartDynamicProvider(c.VA.DNSProvider, 60*time.Second)
+	proto := "udp"
+	if features.Enabled(features.DOH) {
+		proto = "tcp"
```

Comment:
Just confirming that this is the proto value the new SRV records are using? It's not ._doh or something like that?

Reply:
proto is always "udp" or "tcp"